tree/ntuple/v7/src/RNTupleDescriptor.cxx

```diff
@@ -575,6 +576,27 @@ ROOT::Experimental::RResult<void> ROOT::Experimental::Internal::RClusterDescript
    return RResult<void>::Success();
 }
 
+ROOT::Experimental::RResult<void>
+ROOT::Experimental::Internal::RClusterDescriptorBuilder::MarkSuppressedColumnRange(DescriptorId_t physicalId)
+{
+   if (fCluster.fPageRanges.count(physicalId) > 0)
+      return R__FAIL("column ID conflict");
```

Comment:
Given that we don't set a page range for suppressed columns, should this check `fColumnRanges`? In that case, probably also update the check in `CommitColumnRange`.